### ISSUE TITLE
rdkafka queue assertion is firing

### ISSUE THREAD

**Issue by xrl** [fede1024/rust-rdkafka#94]
I was benchmarking a consumer process against a cluster with 20 brokers and a topic with 30 partitions. I disabled all the log statements, so not much to see, but the program crashes after 5 minutes with:

```
$ ./target/release/ridb
*** rdkafka_queue.h:188:rd_kafka_q_destroy0: assert: rkq->rkq_refcnt > 0 ***
```

**Comment by xrl** [comment]
Trying to recreate with `RUST_BACKTRACE=full` and debug symbols.

**Comment by benesch** [comment]
A similar bug was filed against confluent-kafka-go around the same time frame (June, 2018). That bug was closed because it sounds like there's a good chance that things were fixed in librdkafka 1.0. Since we've also bumped past librdkafka v1.0, and in the absence of other details, I'm going to close this out. Please refile another issue if you can reproduce with the latest code, though!
